app/src/main/res/layout/village_typeahead_list_item.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:background="@color/white"
+  android:orientation="vertical">
+
+  <TextView
+    android:id="@+id/villageTypeAheadItemTextView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center_vertical"
+    android:minHeight="?android:attr/listPreferredItemHeightSmall"
+    android:padding="@dimen/spacing_12"
+    android:textAppearance="?android:attr/textAppearanceListItemSmall"
+    tools:text="001 Charni Road" />
+
+  <View
+    android:layout_width="match_parent"
+    android:layout_height="1dp"
+    android:layout_marginHorizontal="@dimen/spacing_12"
```

Comment:
When you use `android:layout_marginHorizontal` it applies margin on `start` and `end`. In the design, the margin is only applied to the start. Can we resolve this, please?

Reply:
Looks like you missed resolving this, can you resolve this in the next PR, please?